src/components/language-selector/language-selector.css

```diff
@@ -22,3 +22,8 @@
     color: #3373cc;
     font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
 }
+
+.language-select:focus {
+    border-color: white;
+    box-shadow: 0 0 0 -2px white;
```

Comment:
Did you mean to have this negative 2 to actually be positive 2? A negative value wouldn't make much sense as a spread radius on a box shadow, it would be the same as having no box shadow at all?

Reply:
I think if all that is needed is a border, there wouldn't need to be box-shadow at all, right?